### PR TITLE
Update community run script

### PR DESCRIPTION
**Pull request by kayeekayee**
run_upp script got the following updates:
-updated directory name from EMC_post to UPP
-updated executable path to tests/install/bin
-added error message for binarynemsiompiio option
-added netcdfpara option

Able to run all 6 cases on Cheyenne with Intel option.

### CODE REVIEW

**Comment by fossell** [comment]
This PR fixes #492

**Comment by fossell** [comment]
@kayeekayee - Have these changes been tested?

File: scripts/run_upp
Comment (hertneky) [inline scripts/run_upp]: With the unification - both GFS and LAM support netcdfpara

File: scripts/run_upp
Comment (hertneky) [inline scripts/run_upp]: What about just removing this part of the if statement altogether?

File: scripts/run_upp
Comment (hertneky) [inline scripts/run_upp:145]: inFormat for LAM can also be netcdfpara

File: scripts/run_upp
Comment (hertneky) [inline scripts/run_upp]: Could consider changing UNIPOST_HOME to UPP_HOME here and throughout.
 (L38, 67, 68, 198-199, 204-205, 208)

**Review by hertneky — CHANGES_REQUESTED** [review]
Some additional mods that couldn't be added in review:

L38 EMC-post -> UPP
L95-97: remove debug comment and extra whitespace
L298 -> LAM add netcdfpara as well

**Comment by kayeekayee** [comment]
> 

@fossell Yes, this changes has been tested on Cheyenne with Intel compiler for all the netcdf and netcdfpara cases. 
Here is the test path: /glade/scratch/kayee/UPP/UFFDA/UFFDA

File: scripts/run_upp
Comment (hertneky) [inline scripts/run_upp]: Formatting here to match the rest of the list

**Review by hertneky — APPROVED** [review]
One super minor formatting change

@fossell Do we know if the GFS model output names are the same with the transition to the global workflow for MRW? The current/old MRW release model output filenames are atmfhhh.nc/sfcfhhh.nc.

**Review by fossell — APPROVED** [review]

**Comment by fossell** [comment]
> @fossell Do we know if the GFS model output names are the same with the transition to the global workflow for MRW? The current/old MRW release model output filenames are atmfhhh.nc/sfcfhhh.nc.

Good question, I can ask and we can update if needed. This PR can go ahead as-is to expedite release testing. We can update again once more information is gathered.

File: scripts/run_upp
Comment (hertneky) [inline scripts/run_upp]: change to flxFileName=${modelDataPath}/phyf${fhour}.nc

Comment (hertneky) [inline scripts/run_upp]: change to flxFileName=${modelDataPath}/phyf${fhour}.nc

**Review by hertneky — CHANGES_REQUESTED** [review]

**Comment by fossell** [comment]
@kayeekayee - What is the status on this testing with these final updates? Has it been run through all cases or just confirmed in a single standalone test?
@hertneky - Can you review the final updates Ka Yee made in response to your requested changes?

**Comment by kayeekayee** [comment]
@fossell I tested 6 cases including netcdfpara with Intel compiler.

**Comment by fossell** [comment]
> @fossell I tested 6 cases including netcdfpara with Intel compiler.

Great! Then I think this should be ok to merge. We'll wait for @hertneky to give her final approval on the PR though.

File: scripts/run_upp
Comment (hertneky) [inline scripts/run_upp:39]: Link on L38 for the UPP Usres Guide does not work - let's get this updated

**Review by hertneky — APPROVED** [review]
I keep finding more things every time I review, but this is the last one, I swear! We can either link the 9.0.1 release or the soon-to-be released 10.0.12, probably the latter

File: scripts/run_upp
Comment (fossell) [reply scripts/run_upp:39]: Good catch. What about changing to https://upp.readthedocs.io/en/latest/ since we control what is "latest" on ReadTheDocs?

Comment (hertneky) [reply scripts/run_upp:39]: That works too. I think the only place latest is linked is on the wiki and points to develop currently.

**Pull request merged**
